ui/app/styles/components/gutter-toggle.scss

```diff
@@ -0,0 +1,19 @@
+.gutter-toggle {
+  display: none;
+
+  position: absolute;
+  left: 0;
+  padding: 1rem;
+  transform: translate(-35%, 0%);
+  height: 100%;
+  fill: desaturate(lighten($nomad-green, 20%), 30%);
+  cursor: pointer;
```

Comment:
Any thoughts on using a style more similar to Terraform's icon they use here? The usage is different, but I like that the icon is pretty minimal

![image](https://user-images.githubusercontent.com/111194/43293320-d7681214-90f6-11e8-8e84-e85e8218a085.png)